mungegithub/mungers/BUILD

```diff
@@ -63,7 +63,7 @@ go_library(
         "cherrypick-must-have-milestone.go",
         "cherrypick-queue.go",
         "cla.go",
-        "close-stale-pr.go",
+        "close-stale-obj.go",
```

Comment:
I will change the name to "close-stale.go", and make corressponding change on the BUILD file.